src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/classes/eltwise_chain.cpp

```diff
@@ -135,6 +135,19 @@ void EltwiseChainTest::SetUp() {
 
         ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwiseOps[eltwiseOps.size() - 1])};
         function = std::make_shared<ov::Model>(results, paramVec, "eltwise_chain_fq");
+    } else if (postNode == PostNode::SOFTSIGN) {
+        std::vector<std::shared_ptr<ov::Node>> eltwiseOps;
+        eltwiseOps.push_back(make_eltwise(inputNodes1[0], inputNodes2[0], eltwiseOpTypes[0]));
+        for (size_t i = 1; i < eltwiseOpTypes.size(); i++) {
+            eltwiseOps.push_back(make_eltwise(eltwiseOps[eltwiseOps.size() - 1], inputNodes2[i], eltwiseOpTypes[i]));
+        }
+
+        auto softsign = ov::test::utils::make_activation(eltwiseOps[eltwiseOps.size() - 1], ElementType::i32, ActivationTypes::SoftSign);
+        eltwiseOps.push_back(softsign);
+
+        ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwiseOps[eltwiseOps.size() - 1])};
+
+        function = std::make_shared<ov::Model>(results, paramVec, "eltwise_chain_softsign");
```

Comment:
Please add an assert `postNode == EMPTY` in "else" block

Reply:
Done

Reply:
The idea behind it is to throw an exception if we add a new `PostNode` type but forget to support it in the builder: we still don't have such behavior. Let's introduce "else" section where we place smth like `OPENVINO_THROW("Unsupported postNode type");`